src/main/java/com/openshift/jenkins/plugins/pipeline/MessageConstants.java

```diff
@@ -0,0 +1,60 @@
+package com.openshift.jenkins.plugins.pipeline;
+
+public class MessageConstants {
+
+public static final String CANNOT_GET_CLIENT = "\n\nExiting \"%s\" unsuccessfully; a client connection to \"%s\" could not be obtained.";
+public static final String BAD_BUILD_STATE = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [%s].";
+public static final String GOOD_BUILD_WITH_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].  All deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_BAD_DEPLOY = "\n\nExiting \"%s\" unsuccessfully; build \"%s\" has completed with status:  [Complete]. However, not all deployments with ImageChange triggers based on this build's output triggered off of the new image.";
+public static final String GOOD_BUILD_IGNORE_DEPLOY = "\n\nExiting \"%s\" successfully; build \"%s\" has completed with status:  [Complete].";
+public static final String START_BUILD_CANCEL = "\n\nStarting the \"%s\" action for build config \"%s\" from the project \"%s\".";
```

Comment:
ah ... the message is not as generic as it might imply (though the distinction may be too subtle);  the cancel build/deploy are "post-build actions" and the rest of them are "build steps" per jenkins UI terminology; so you see "action" in this message, and you see "step" in others.

I did consider using parameters for "action" vs. "step" and achieve some more message consolidation (there were also some opportunities to plugin in "deploy" vs. "build" for the config) ... gotta admin just got tired of string editing.

I can circle back and do that if you want.  Then the string constant name could be more generic as well.